playwright.config.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+const CI = false;
+
+module.exports = {
+  testDir: "./test/e2e-playwright",
+  // however this can have benefits, in tests I couldn't start a server on multiple ports
+  // for tests "stats-refactored.test.js" the port is read from "ports-map.js" file somehow
+  // I should manage to handle it if I want to run the tests in parallel mode
+  fullyParallel: false,
+  forbidOnly: !!CI,
+  retries: CI ? 2 : 0,
+  workers: 1,
+  reporter: "html",
```

Comment:
We don't need HTML report when running tests on CI

Reply:
you are right, while I was checking the playwright documentations, I noticed there is an option for Github Action annotations, so I have set it to report to Github Action in case its running in CI, otherwise use list format.